modules/ROOT/pages/amazon/amazon-sqs-connector-studio.adoc

```diff
@@ -0,0 +1,90 @@
+. Follow the prompts to install the connector.
+
+=== Install the Connector from the Mule Palette
+
+. In Studio, create a Mule project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the `Amazon SQS` in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source
+
+Configure an input source for the connector such as a connector operation, *HTTP Listener*, or *Scheduler*.
+
+== Configure the Connector
+
+To use Amazon SQS Connector in your Mule application, configure a global element that can be used by all Amazon SQS Connectors in the application.
+
+. Drag a connector operation onto the Studio canvas.
+. To create a global element for the connector, click the *Global Elements* tab at the base of the canvas, and then click *Create*.
+. In the *Choose Global Type* window, expand *Connector Configuration*, and select *Amazon SQS Configuration*:
++
+image::amazon/amazon-sqs-studio-select-global.png[Select Connector for Global Element]
++
+. Click *OK*.
+. Specify the global element properties.
++
+You can either enter your credentials into the global configuration properties or reference a configuration placed in a file in the `src` folder of your project file that contains these values. For simpler maintenance and better reuse of your project, use a configuration file.
++
+For example:
++
+image::amazon/amazon-sqs-studio-global-config.png[Configure Global Element]
++
+. Configure the parameters as follows:
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Name | Name used to reference the configuration
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the account
+|Secret Key |Key that plays the role of a password
+|Try AWS Credentials Provider Chain|Dropdown that controls whether temporary credentials should be used
+|Region Endpoint |The regional endpoint to process your requests
+|Default Global URL |URL of the Amazon SQS queue to act upon
+|===
++
+When you provide a queue URL in the *Default Global Queue URL* field of the global element, the connector automatically creates the queue and sets the URL of this queue as the provided value. All Amazon SQS Message processors that reference the global element perform operations using this queue URL.
++
+To reference a different queue URL for a particular message processor in the flow, perform the operation using the `Queue URL` attribute provided by the message processor.
+. Leave the default entries for the *Proxy* and *Advanced* tabs.
+. Click *Test Connection* to confirm that the parameters of your global configuration are accurate, and that Mule is able to connect successfully to your instance of Amazon SQS.
+. Click *OK*.
+
+== Next Step
+
+After configuring this connector in Studio, see the
+xref:amazon/amazon-sqs-connector-config-topics.adoc[Additional Configuration Information]
+topic for more configuration steps.
+
```

Comment:
I am still not convinced about the logic of having both a Configure topic and an "Additional Configuration Information" topic. I wonder what someone scanning the TOC thinks that that title even means: is it optional?

Reply:
The template is a work in progress and I am finding it does not work well for the very complex connectors, like the new WebSockets connector. I agree that we should revisit this and maybe split out the Additional Configuration information in separate topics if the material lends itself to that.